DESCRIPTION

```diff
@@ -1,7 +1,8 @@
 Package: goodpractice
 Title: Advice on R Package Building
-Version: 1.0.0
-Author: Gabor Csardi
+Version: 1.0.1
```

Comment:
Version bumps shouldn't be a problem but if you want to keep it the same, I'll leave it as is.